hera_pspec/pspecdata.py

```diff
@@ -858,7 +910,7 @@ def cov_q_hat(self, key1, key2, time_indices=None):
 
         Returns
         -------
-        cov_q_hat: matrix with covariances between un-normalized band powers
+        cov_q_hat: matrix with covariances between un-normalized band powers (Ntimes, Nfreqs, Nfreqs)
```

Comment:
This breaks readthedocs -- description should be indented, on the second line, e.g.
```
cov_q_hat: array_like
    Matrix with covariances between un-normalized band powers (Ntimes, Nfreqs, Nfreqs)
```

Reply:
fixed.